app/javascript/packs/widget.js

```diff
@@ -10,6 +10,8 @@ import i18n from '../widget/i18n';
 import { isPhoneE164OrEmpty } from 'shared/helpers/Validators';
 import router from '../widget/router';
 import { domPurifyConfig } from '../shared/helpers/HTMLSanitizer';
+import PhoneInput from '../widget/components/Form/PhoneInput';
```

Comment:
@iamsivin Can we import this only when needed. This component gets loaded when the initial package is loaded. I don't think we need to import this piece if there is no pre-chat form.